Install with Python 3.11 fails

As part of the move to RHEL9 based system for ITER's SDCC, I've tried to install `cherab.imas` with Python 3.11. It complained with the following cython-related error:
```
      running build_ext
      building 'cherab.imas.math.functions.vector_functions' extension
      gcc -DNDEBUG -g -fwrapv -O3 -Wall -O2 -ftree-vectorize -march=sandybridge -fno-math-errno -fPIC -O2 -ftree-vectorize -march=sandybridge -fno-math-errno -fPIC -fPIC -I. -I/tmp/pip-build-env-iptcz_k7/overlay/lib/python3.11/site-packages/numpy/core/include -I/work/imas/opt/EasyBuild/software/Python/3.11.5-GCCcore-13.2.0/include/python3.11 -c cherab/imas/math/functions/vector_functions.c -o build/temp.linux-x86_64-cpython-311/cherab/imas/math/functions/vector_functions.o
      cherab/imas/math/functions/vector_functions.c:296:12: fatal error: longintrepr.h: No such file or directory
        296 |   #include "longintrepr.h"
            |            ^~~~~~~~~~~~~~~
      compilation terminated.
      error: command '/work/imas/opt/EasyBuild/software/GCCcore/13.2.0/bin/gcc' failed with exit code 1
      [end of output]
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
  ERROR: Failed building wheel for cherab-imas
Failed to build cherab-imas
ERROR: Could not build wheels for cherab-imas, which is required to install pyproject.toml-based projects
```

It seems like avoiding a pre-release cython3 helps, e.g. with the following patch to `pyproject.toml`:
```
$ git diff pyproject.toml
diff --git a/pyproject.toml b/pyproject.toml
index 5444a93..39f8695 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -1,3 +1,3 @@
 [build-system]
-requires = ["setuptools", "wheel", "oldest-supported-numpy", "cython==3.0a5", "raysect==0.7.1", "cherab==1.4"]
+requires = ["setuptools", "wheel", "oldest-supported-numpy", "cython>=3", "raysect==0.7.1", "cherab==1.4"]
 build-backend="setuptools.build_meta"
```

--- a/README.md
+++ b/README.md
@@ -8,16 +8,20 @@ This module enables the creation of Cherab plasma objects and Raysect meshes fro
 
 This add-on module requires IMAS to be installed with Python interface. 
 
-On ITER SDCC load the respective modules:
+Imnstall with the following commands.
+
+On ITER SDCC CentOS8 nodes:
 
 ```bash
 module load IMAS
 module load Raysect/0.7.1-intel-2020b
+pip install <path-to-cherab-imas> --user
 ```
-and then install with:
+
+On ITER SDCC RHEL9 nodes:
 
 ```bash
-pip install cherab==1.4.0 --user
-pip install -U cython==3.0a5 --user
+module load IMAS
+module load Raysect/0.7.1-iimkl-2023b
 pip install <path-to-cherab-imas> --user
 ```

--- a/pyproject.toml
+++ b/pyproject.toml
@@ -1,3 +1,3 @@
 [build-system]
-requires = ["setuptools", "wheel", "oldest-supported-numpy", "cython==3.0a5", "raysect==0.7.1", "cherab==1.4"]
+requires = ["setuptools", "wheel", "oldest-supported-numpy", "cython>=3", "raysect==0.7.1", "cherab==1.4"]
 build-backend="setuptools.build_meta"

